test/Microsoft.NET.Build.Tests/GivenThatWeWantToResolveConflicts.cs

```diff
@@ -320,8 +320,8 @@ public void PlatformPackagesCanBePruned(bool prunePackages)
         [InlineData("net6.0")]
         [InlineData("netcoreapp3.1")]
         [InlineData("netcoreapp3.0")]
-        [InlineData("netcoreapp2.1", false)] //TODO: https://github.com/dotnet/sdk/issues/49917
-        [InlineData("netcoreapp2.0", false)] //TODO: https://github.com/dotnet/sdk/issues/49917
```

Comment:
Any reason not to include `netcoreapp2.2` or is that covered because 2.1 has the same references?

Reply:
Yes, low risk that something would be broken specifically for .NET Core 2.2, and it's way out of support anyway.